### DISCUSSION TITLE
Add a minimize feature

### DISCUSSION THREAD

**Discussion by bryanflynn2017** [nefarius/HidHide#161] [Ideas]
This program does exactly what I've needed for a long time, the only problem is that it's always on my desktop. I'm not sure why there isn't a minimize button already.

**Comment by nefarius** [comment]
Why would there? Typical use-case is people configure it once, close the GUI and forget. Why leave it running? Also leaving it open permanently blocks other apps that interface with HidHide, so not a swell idea overall.

**Reply by bryanflynn2017** [reply]
My bad, I didn't know the program's changes would act that way, thought it needed to be runnning. Thank you for the very quick reply!

**Reply by nefarius** [reply]
No worries, also, if you need to make frequent changes, I recommend you use the shipped command line app and make some scripts.
